registry/templates/donor/award_document.html

```diff
@@ -28,7 +28,7 @@ <h1>
             </p>
             <p>
                 S potěšením Vám sdělujeme, že mezi vyznamenanými {{ medal_title }}
-                <br> prof. MUDr. J. Jánského je pracovník Vašeho podniku
+                prof.&nbsp;MUDr.&nbsp;J.&nbsp;Jánského je pracovník Vašeho podniku
```

Comment:
@adelpopelkova Why did you use three times `nbsp`, I think that only one is enough. I don't see any differents if I've used last two `nbsp` in my browser.

Reply:
Nbsp should be used anywhere where you want the words around them to always be together without an automatic line break between them. If for any reason, the paragraph gets bigger, with these nbsps you can be sure that the professor's name will always be on one line.

Reply:
aa ok